### ISSUE TITLE
too many redirects  - multi site network

### ISSUE THREAD

**Issue by mysitesrock** [phikai/wordpress-force-http#1]
Hi, I recently had to deactivate your plugin due to it causing an error - too many redirects for one of my sites / subfolders.

Running WP: 4.4.1
WP Force HTTP: Version 0.1.2

It works fine until I change a site's settings to become a website vs. blog.  Etc - set a static page to home page. Blog page to a blog page.

network in question. Sites.Berry.edu
site discovered upon. sites.berry.edu/tle

**Comment by phikai** [comment]
It looks like that site has some kind of plugin or configuration that forces it to HTTPS on the public side. So when my plugin tries to send it back to regular HTTP, they get in a redirect fight causing the infinite loop.

```
➜  ~  curl -I http://sites.berry.edu/tle/
HTTP/1.1 301 Moved Permanently
Date: Tue, 26 Jan 2016 15:12:16 GMT
Server: Apache/2.4.7 (Ubuntu)
X-Powered-By: PHP/5.5.9-1ubuntu4.14
Set-Cookie: wfvt_1158063328=56a78cd15a2be; expires=Tue, 26-Jan-2016 15:42:17 GMT; Max-Age=1800; path=/; httponly
Location: https://sites.berry.edu/tle/
Content-Type: text/html; charset=UTF-8

➜  ~  curl -I https://sites.berry.edu/tle/
HTTP/1.1 200 OK
Date: Tue, 26 Jan 2016 15:12:23 GMT
Server: Apache/2.4.7 (Ubuntu)
X-Powered-By: PHP/5.5.9-1ubuntu4.14
Set-Cookie: wfvt_3864273964=56a78cd7aee25; expires=Tue, 26-Jan-2016 15:42:23 GMT; Max-Age=1800; path=/; httponly
Link: <https://sites.berry.edu/tle/wp-json/>; rel="https://api.w.org/"
Link: <https://sites.berry.edu/tle/>; rel=shortlink
Content-Type: text/html; charset=UTF-8
```

**Comment by mysitesrock** [comment]
Would it be because I have this in the wp-config?

define('FORCE_SSL_ADMIN', true);


I'm thinking about just making the entire site SSL.


On Tue, Jan 26, 2016 at 10:12 AM, A. Kai Armstrong <notifications@github.com

**Comment by phikai** [comment]
No. It would be because something on that site has it set to force HTTPS. Probably another plugin configured just for that site and not the other network.

If you're going to make your entire site HTTPS, this may be an option:
 https://github.com/phikai/wordpress-force-https

**Comment by mysitesrock** [comment]
Awesome... appreciate it!
